closed/autoconf/custom-hook.m4

```diff
@@ -332,6 +333,79 @@ AC_DEFUN([OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU],
   esac
 ])
 
+AC_DEFUN([OPENJ9_CONFIGURE_CRIU_SUPPORT],
+[
+  AC_MSG_CHECKING([for CRIU support])
+  AC_ARG_ENABLE([criu-support], [AS_HELP_STRING([--enable-criu-support], [enable CRIU support @<:@disabled@:>@])])
+  OPENJ9_ENABLE_CRIU_SUPPORT=false
+
+  if test "x$enable_criu_support" = xyes ; then
+    AC_MSG_RESULT([yes (explicitly enabled)])
+    OPENJ9_ENABLE_CRIU_SUPPORT=true
+  elif test "x$enable_criu_support" = xno ; then
+    AC_MSG_RESULT([no (explicitly disabled)])
+  elif test "x$enable_criu_support" = x ; then
+    case "$OPENJ9_PLATFORM_CODE" in
+      xa64|xl64|xr64|xz64)
+        AC_MSG_RESULT([yes (default)])
+        OPENJ9_ENABLE_CRIU_SUPPORT=true
+        ;;
+      *)
+        AC_MSG_RESULT([no (default)])
+        ;;
+    esac
+  else
+    AC_MSG_ERROR([--enable-criu-support accepts no argument])
+  fi
+  AC_SUBST(OPENJ9_ENABLE_CRIU_SUPPORT)
+])
```

Comment:
With this, there are now two copies of `OPENJ9_CONFIGURE_CRIU_SUPPORT`.

Reply:
Will fix it

Reply:
Created https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/711